pwiz_tools/Skyline/Model/Lib/Midas/MidasLibrary.cs

```diff
@@ -294,7 +294,15 @@ private bool Load(IProgressMonitor monitor)
             _spectra = null;
             if (FilePath == null)
                 return false;
-            var info = new FileInfo(FilePath);
+            FileInfo info;

```

Comment:
How exactly are you expecting the user to see issues with their Midas library? It seems like all error information is being swallowed and not clearly reported to the user. Even if the eventual result is to report that loading the file failed, it seems like the exception, in this case, might have more information about why.